e2e/components/toolbar/toolbar.ts

```diff
@@ -62,7 +62,10 @@ export class Toolbar extends Component {
   }
 
   async isButtonPresent(title: string) {
-    return this.byCss(`button[title="${title}"]`).isPresent();
+    const element = this.byCss(`button[title="${title}"]`);
+    await waitForPresence(element)
```

Comment:
from a logical point of view, this is incorrect: it's waiting for presence, and then returning isPresent.
this will never return false if the item is not present, but it will throw an error

Reply:
please remove this waitForPresence